packages/fxa-content-server/tests/server/routes/get-index.js

```diff
@@ -86,6 +86,8 @@ registerSuite('routes/get-index', {
                 appDisplayVersion: sentConfig.release,
                 channel: serversideGleanConfig.appChannel,
                 serverEndpoint: serversideGleanConfig.serverEndpoint,
+                logPings: serversideGleanConfig.logPings,
```

Comment:
nit: Technically serverside is not a word, so a better name would be serverSideGleanConfig. (better cause it doesn't set off my code spell checker, ;p)

Reply:
haha... one of those things where I make the mistake once and the copied it all over the place.  And I don't possess a code spell checker.